data/data-pipeline/tile/generate.py

```diff
@@ -24,31 +25,30 @@ def generate_tiles(data_path: Path) -> None:
     # generate high mbtiles file
     logger.info(f"Generating USA High mbtiles file")
     cmd = "tippecanoe "
-    cmd += "--force -Z 5 -z 11 -l usa_high "
-    cmd += "-o " + str(high_tile_path) + "/usa_high.mbtiles "
+    cmd += "--minimum-zoom=5 --maximum-zoom=11 --layer=blocks "
+    cmd += "--output=" + str(high_tile_path) + "/usa_high.mbtiles "
     cmd += str(score_geojson_dir / "usa-high.json")
-    os.system(cmd)
+    call(cmd, shell=True)
 
     # generate high mvts
     logger.info(f"Generating USA High mvt folders and files")
     cmd = "tippecanoe "
-    cmd += " --drop-densest-as-needed --no-tile-compression  -zg "
-    cmd += "-e " + str(high_tile_path) + " "
+    cmd += "--minimum-zoom=5 --maximum-zoom=11 --no-tile-compression "
+    cmd += "--output-to-directory=" + str(high_tile_path) + " "
     cmd += str(score_geojson_dir / "usa-high.json")
-    os.system(cmd)
+    call(cmd, shell=True)
 
     # generate low mbtiles file
     logger.info(f"Generating USA Low mbtiles file")
     cmd = "tippecanoe "
-    cmd += "--force -Z 5 -z 11 -l usa_low "
-    cmd += "-o " + str(low_tile_path) + "/usa_low.mbtiles "
+    cmd += "--minimum-zoom=0 --maximum-zoom=7 --layer=blocks "
+    cmd += "--output=" + str(low_tile_path) + "/usa_low.mbtiles "
     cmd += str(score_geojson_dir / "usa-low.json")
-    os.system(cmd)
+    call(cmd, shell=True)
 
     # generate low mvts
     logger.info(f"Generating USA Low mvt folders and files")
-    cmd = "tippecanoe "
-    cmd += " --drop-densest-as-needed --no-tile-compression  -zg "
-    cmd += "-e " + str(low_tile_path) + " "
+    cmd = "tippecanoe --minimum-zoom=0 --maximum-zoom=7 --no-tile-compression "
```

Comment:
might make sense for min and max zoom for low/high to be constants, as they're repeated a few times in this file - easier to change if we need to alter in the future / easier to keep mvt and mbtiles in sync